### PR TITLE
CMS Data Updater

### PR DESCRIPTION
**Pull request by mikelehen**

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -5,3 +5,5 @@ data/cases-cds/timeseries.csv.zip filter=lfs diff=lfs merge=lfs -text
 data/cases-covid-county-data/timeseries-common.csv filter=lfs diff=lfs merge=lfs -text
 data/covid-tracking/*.csv filter=lfs diff=lfs merge=lfs -text
 data/covid-tracking/*.json filter=lfs diff=lfs merge=lfs -text
+data/testing-cms/archive/** filter=lfs diff=lfs merge=lfs -text
+data/testing-cms/timeseries-common.csv filter=lfs diff=lfs merge=lfs -text
```
Comment (mikelehen) [inline .gitattributes:9]: FWIW- I don't really know what our criteria is for putting things in LFS.  This amounts to 4.2M right now, mostly in the archive/ folder which will grow over time.

I was less sure about the actual CSV (it's 1.5MB).

FWIW- I didn't put the HHS data into LFS, but it's only a 500k CSV. 🤷

Comment (ghop02) [reply .gitattributes:9]: I would prefer to include all csv files in LFS to be consistent - but not an immediate priority.  I made [this ticket](https://trello.com/c/8dJBzupQ/520-add-all-data-files-to-git-lfs) to standardize

File: scripts/update_cms_testing_data.py [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+
+def read_cms_dataset_from_zip(zip_path: pathlib.Path) -> pd.DataFrame:
+    """Finds and reads the Excel file within the CMS dataset zip file."""
+    zip = zipfile.ZipFile(zip_path)
+    excel_files = [
+        f for f in zip.filelist if f.filename.endswith(".xlsx") and "__MACOSX" not in f.filename
+    ]
+    assert len(excel_files) == 1
+
+    for headers in range(5, 8):
```
Comment (ghop02) [inline scripts/update_cms_testing_data.py:122]: what does range 5-8 indicate?

Comment (ghop02) [reply scripts/update_cms_testing_data.py:122]: Not sure I totally understand what this section is doing

**Review by ghop02 — COMMENTED** [review]
Overall looks good! Couple small comments

**Comment by ghop02** [comment]
Oh also, doyou need to add beautiful soup to requirements.txt?

File: scripts/update_cms_testing_data.py (hunk 2 above)
Comment (mikelehen) [reply scripts/update_cms_testing_data.py:133]: Oh, whoops. I swear I had a comment explaining this, but I must have lost it while moving code around.  Added a comment to explain this (admittedly hacky) code:

```
# HACK: The excel file has some "header" rows at the top before the columns are defined.
# Unfortunately, the number of header rows has changed over time, and so we don't know how
# many there will be. So we try parsing the file repeatedly with different numbers of header
# rows until we successfully parse out a "County" column, which should always be present.
```

**Review by mikelehen — COMMENTED** [review]
Thanks for the review! Finally got back to this and made the requested changes.

FYI- I already added beautiful soup to requirements.txt.

**Comment by mikelehen** [comment]
@ghop02 Can you give this another look (and an approval) when you get a chance?

**Review by ghop02 — APPROVED** [review]
LGTM!

**Pull request merged**